config/cmake/H5pubconf.h.in

```diff
@@ -319,12 +307,6 @@
    compiled */
 #cmakedefine H5_HAVE_ROS3_VFD @H5_HAVE_ROS3_VFD@
 
-/* Define to 1 if you have the `round' function. */
-#cmakedefine H5_HAVE_ROUND @H5_HAVE_ROUND@
-
-/* Define to 1 if you have the `roundf' function. */
-#cmakedefine H5_HAVE_ROUNDF @H5_HAVE_ROUNDF@
-
 /* Define to 1 if you have the `setjmp' function. */
```

Comment:
We just have to be aware that these symbols might go missing on some platforms and raise binary compatibility issues.

Reply:
> We just have to be aware that these symbols might go missing on some platforms and raise binary compatibility issues.

Reverted